spec/storey/create_plain_schema_spec.rb

```diff
@@ -4,7 +4,7 @@
   context 'when there is no suffix set' do
     it 'should create a schema without a suffix' do
       Storey.create_plain_schema 'dun'
-      Storey.schemas(suffix: true).should include('dun')
+      expect(Storey.schemas(suffix: true)).to include('dun')
```

Comment:
Should this be `suffix: false`?

Reply:
I think that's correct. There's no suffix set in `Storey.suffix` (which should be `Storey.configuration.suffix` -- will fix that).